openstackquery/time_utils.py

```diff
@@ -47,3 +47,10 @@ def convert_to_timestamp(
         return datetime.fromtimestamp(current_time - time_in_seconds).strftime(
             "%Y-%m-%dT%H:%M:%SZ"
         )
+
+    @staticmethod
+    def get_uptime(uptime_string: str):
```

Comment:
```suggestion
    def get_uptime(uptime_string: str) -> str:
```

Reply:
That's my bad, I was testing things this isn't being used

Reply:
No worries, that's my fault on being too eager to review haha